src/subject/commands/end-responsability-command.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Injectable,
+} from '@nestjs/common';
+import { PrismaService } from 'src/database/prisma.service';
+import { AlreadyFinishedException, BlockingMonitorsException, ResponsabilityNotFoundException } from '../utils/exceptions';
+import { SubjectResponsabilityStatus } from '../utils/subject.enum';
+import { MonitorStatus } from 'src/monitor/utils/monitor.enum';
+
+@Injectable()
+export class EndResponsabilityCommand {
+  constructor(private prisma: PrismaService) {}
+
+  async execute(responsabilityId: number): Promise<void> {
+    // Offset em milissegundos
+    const offsetAMT = -4 * 60 * 60 * 1000; 
```

Comment:
Se deixar o nome de variável mais descritivo não precisa do comentário:

```suggestion
    const AMT_OFFSET_IN_MS = -4 * 60 * 60 * 1000; 
```

PS.: Coloco em letras maiúsculas por ser uma constante simbólica. **[Mais detalhes aqui](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/)**

Reply:
Thanks! Done